src/apps/company-lists/client/CreateListForm.jsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FieldInput } from '../../../client/components'
+import TaskForm from '../../../client/components/Task/Form'
+
+const CreateListForm = (
+  {
+    id,
+    name,
+    hint,
+    label,
+    cancelUrl,
+    maxLength,
+    csrfToken,
+  }) => {
+
+  return (
+    <TaskForm
+      id="create-list-form"
+      submissionTaskName="Create list"
+      analyticsFormName="create-list-form"
```

Comment:
```suggestion
      analyticsFormName="createListForm"
```

Also this needs to be snake case